### ISSUE TITLE
MD format for notes

### ISSUE THREAD

**Issue by snehasharma1111** [akshatmittal61/planner#26]
**Is your feature request related to a problem? Please describe.**
In the notes window, the user cannot yet customize or format their notes, with headings, links etc.

**Describe the solution you'd like**
The user should be able to use, bold or italic text, images, links, etc in their notes.

**Describe alternatives you've considered**
W can add Markdown support to support all these features.

**Additional context**
![image](https://user-images.githubusercontent.com/92130285/195229020-4bfd72a2-86f6-410d-a571-6641c752a6a7.png)

**Comment by akshatmittal61** [comment]
Yes, that seems like a great idea, can you please raise a PR by showcasing your solution?
Your changes will be approved after a review.
